src/effects/effectsmanager.h

```diff
@@ -37,7 +37,7 @@ class EffectsManager {
     EffectChainPointer getOutputEffectChain() const;
 
     EngineEffectsManager* getEngineEffectsManager() const {
-        return m_pEngineEffectsManager;
+        return m_pEngineEffectsManager.get();
     }
```

Comment:
can you add a comment to document the lifetime of the returned pointer?

Reply:
I have added a comment. However, that is not very safe. All the memory management of the Effect engine can make use of an overhaul, but not in this PR.